src/app/App.js

```diff
@@ -26,6 +27,19 @@ export const App = ({ options }) => {
   });
   const mainCointainer = document.getElementById('mainContainer');
   renderRedButton(mainCointainer);
+
+  // Modal window
+  const modalWindow = new ModalWindow();
+
+  const testButtonForWindow = document.getElementById('btnForModalWindow');
+
+  testButtonForWindow.onclick = function () {
+    const testContent = document.createElement('div');
+    testContent.innerText = 'dsadasdsadas';
+    modalWindow.show(testContent);
```

Comment:
Zastanawiam się czy show powinno być w tym miejscu czy też w ModalWindow(). Chyba bardziej pasowałoby, żeby to określać w konstruktorze, a potem nie musieć się martwić?

Reply:
No ok, ale jak będzie w konstruktorze, to wtedy za każdym razem kiedy musimy tworzyć obiekt, żeby przekazać mu content do wyświetlenia, tak? 
typu:
new ModalWindow(parentElement, contentToDisplay)

Reply:
No wtedy tak. `new ModalWindow(contentToDisplay)`. Tego parenta tutaj nie widziałem u Ciebie. 
Akurat tutaj mamy jeden modal, ale chyba nibylby problem zrobic kilka? Np. teraz nie jestes w stanie wyswietlic np. 1 modala na 2. Moze taka byc sytuacja - wymyslajac troche.

Reply:
Kiedy tylko można to unikamy modyfikacji obiektów po utworzeniu.